backend/experiment/admin.py

```diff
@@ -349,6 +356,55 @@ def dashboard(self, request, obj, parent_obj=None):
             },
         )
 
+    def remarks(self, obj):
+        remarks_array = []
+
+        # Check if there is any translated content
+        content = obj.translated_content.first()
+
+        if not content:
+            remarks_array.append(
+                {
+                    "level": "warning",
+                    "message": "📝 No transl. content",
+                    "title": "Please add at least one translated content to this experiment.",
+                }
+            )
+
+        has_consent = obj.translated_content.filter(consent__isnull=False).exclude(consent="").first()
+
+        if not has_consent:
+            remarks_array.append(
+                {
+                    "level": "info",
+                    "message": "📋 No consent form",
+                    "title": "Please add at least one consent form to this experiment.",
```

Comment:
It's not always desired for consent forms to be present (e.g., when users come in from external links with a `participant_id`).
```suggestion
                    "title": "You may want to add a consent form (approved by an ethical board) to this experiment.",
```